security-policies/bundle/compliance/policy/azure/data_adapter.rego

```diff
@@ -105,8 +105,8 @@ is_network_watchers_flow_log if {
 	input.subType == "azure-network-watchers-flow-log"
 }
 
-is_network_watcher if {
-	input.subType == "azure-network-watcher"
+is_batched_network_watcher_by_location if {
+	input.subType == "azure-batched-network-watchers-by-location"
 }
```

Comment:
I suggest to revert this as we typically don't follow this naming scheme for sub-types AFAICT. E.g. `azure-bastion` is a batched type but it doesn't have such a long name. We could make a decision to rename those but I think we should be consistent for now.

Reply:
But isn't just a network watcher either. I find the `azure-bastion` without any hint that it's a special type of asset misleading

Reply:
I slept over it and I'll change to `azure-network-watcher`.

I still think it would be a good idea to have a way to hint case the resource is not a traditional resource